src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -353,9 +355,9 @@ public sealed partial class FileSystem
         public static void FileGet(int FileNumber, ref int Value, long RecordNumber = (long)-1) { }
         public static void FileGet(int FileNumber, ref long Value, long RecordNumber = (long)-1) { }
         public static void FileGet(int FileNumber, ref float Value, long RecordNumber = (long)-1) { }
-        public static void FileGet(int FileNumber, ref string Value, long RecordNumber = (long)-1, bool StringIsFixedLength = false) { }
-        public static void FileGet(int FileNumber, ref System.ValueType Value, long RecordNumber = (long)-1) { }
-        public static void FileGetObject(int FileNumber, ref object Value, long RecordNumber = (long)-1) { }
+        public static void FileGet(int FileNumber, [System.Diagnostics.CodeAnalysis.AllowNull] ref string Value, long RecordNumber = (long)-1, bool StringIsFixedLength = false) { }
+        public static void FileGet(int FileNumber, [System.Diagnostics.CodeAnalysis.AllowNull] ref System.ValueType Value, long RecordNumber = (long)-1) { }
```

Comment:
Is this needed because a ValueType can be, Nullable<T>?

Reply:
From a cursory inspection, it looked like the implementation didn't care about the state of the value as an input and is just assigning to it, like an out.  Maybe I'm overthinking it though; I can remove the annotation.  I think the number of C# uses of this method is and will be 0 :)

Reply:
haha, then whichever you prefer 😄